python/pyspark/sql/tests.py

```diff
@@ -847,6 +847,14 @@ def test_input_file_name_reset_for_rdd(self):
         for result in results:
             self.assertEqual(result[0], '')
 
+    def test_input_file_name_udf(self):
```

Comment:
The conflict was here. I didn't run this test so we should see in Jenkins.

Reply:
Got it~